tests/test.spec.ts

```diff
@@ -0,0 +1,14 @@
+import { test, expect } from '@playwright/test';
+test('my first test', async ({ page }) => {
+  
+  test.skip(!!process.env.CI, 'Test file is skipped in CI due to Cloudflare protection.');
+  await page.goto('https://practicesoftwaretesting.com/auth/login');
+  await page.locator('id=email').fill('customer@practicesoftwaretesting.com');
+  await page.locator('id=password').fill('welcome01');
+  await page.locator('data-test=login-submit').click();
+  await expect(page).toHaveURL('https://practicesoftwaretesting.com/account');
```

Comment:
And here as well, you won't have to specify the full link once you add base URL to the config. Here you can leave only remaining part as RegExp
```suggestion
  await expect(page).toHaveURL(/\/account/);
```